src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -111,9 +110,17 @@ protected override void ContextUnloaded(ExecutionContext context)
             }
         }
 
-        public static ApplicationEngine Create(TriggerType trigger, IVerifiable container, StoreView snapshot, long gas, bool testMode = false)
-            => applicationEngineProvider?.Create(trigger, container, snapshot, gas, testMode)
-                ?? new ApplicationEngine(trigger, container, snapshot, gas, testMode);
+        public static long TestGas(StoreView snapshot = null)
```

Comment:
Just add a big enough constant? We don't need a function.

Reply:
Without the snapshot? 9000 of Gas?